coreprofile/10/_index.md

```diff
@@ -1,21 +1,40 @@
 ---
-title: "Jakarta EE Core Profile 10 (under development)"
-date: 2021-04-07
+title: "Jakarta EE Core Profile 10"
+date: 2022-05-30
 summary: "Release of the Jakarta EE Core Profile 10"
 ---
 The Jakarta EE Core Profile defines a profile of the Jakarta EE platform specifically targeted at smaller runtimes.
 
-The goals for this release are:
+The goals for this release were:
 1. To provide a profile that contains a set of Jakarta EE Specifications targeting smaller runtimes suitable for microservices and ahead-of-time compilation.
-2. The TCK(s) for the Core profile should be possible to run independently of the Full Platform TCK.
+2. The TCK(s) for the Core profile run independently of the Full Platform TCK.
 3. Java SE version will be aligned with the Platform Project's decision regarding Java SE versions.
 
+# Artifacts
 
 * [Jakarta EE Core Profile 10 Release Record](https://projects.eclipse.org/projects/ee4j.jakartaee-platform/releases/core-profile-10)
-
+    * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
+* [Jakarta Core Profile 10.0 Specification Document](jakarta-coreprofile-spec-10.0.pdf) (PDF)
+* [Jakarta Core Profile 10.0 Specification Document](jakarta-coreprofile-spec-10.0.html) (HTML)
+* [Jakarta Core Profile 10.0 Javadoc](./apidocs)
+* [Jakarta Core Profile 10.0 TCK](https://download.eclipse.org/jakartaee/coreprofile/10/jakarta-coreprofile-10.0.0.zip)
+  ([sig](https://download.eclipse.org/jakartaee/coreprofile/10/jakarta-coreprofile-10.0.0.zip.sig),
+  [sha](https://download.eclipse.org/jakartaee/coreprofile/10/jakarta-coreprofile-10.0.0.zip.sha256),
+  [pub](https://jakarta.ee/specifications/jakartaee-spec-committee.pub))
+    * TCK Summary Results [WildFly 27.0.0.Alpha1](https://github.com/jakartaredhat/wf-core-tck-runner/wiki/Jakarta-Core-Profile-10.0.0-TCK-Results)
```

Comment:
This was the line that should be yanked.  Ok to leave the ratified mention as that's the format we chose (though not my personal preference)

Reply:
Ah ok. I followed the 9.1 format exactly in the latest update.